src/simplewallet/simplewallet.cpp

```diff
@@ -2074,6 +2132,11 @@ bool simple_wallet::stop_mining(const std::vector<std::string>& args)
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool simple_wallet::set_node(const std::vector<std::string>& args)
+{
+    return prompt_for_new_node();
```

Comment:
Why is it necessary to make another function `prompt_for_new_node` which is called only from here? Why not put the function body directly in `set_node`?